### PR TITLE
feat: Nervos to Ethereum cross-chain contract

### PR DESCRIPTION
**Pull request by huwenchao**

### CODE REVIEW

File: eth-contracts/contracts/AssetManager.sol [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            require(tokenAddr != address(0), "Asset not found");
+            IMirrorToken(tokenAddr).mint(records[i].to, records[i].amount);
+            emit Mint(records[i].assetId, tokenAddr, records[i].to, records[i].amount, records[i].lockId);
+        }
+    }
+
+    /// @dev Burn an asset and get the locked amount back on Nervos chain.
+    /// @param token The address of the token to burn.
+    /// @param amount The amount to burn.
+    function burn(address token, uint256 amount) public nonReentrant {
```
Comment (mkxbl) [inline eth-contracts/contracts/AssetManager.sol:62]: forget to handle bridge fee?

Comment (huwenchao) [reply eth-contracts/contracts/AssetManager.sol:62]: It's an offchain option, check:
https://github.com/nervosnetwork/force-bridge/blob/main/ckb-contracts/types/schemas/recipient_typescript.mol

Comment (mkxbl) [reply eth-contracts/contracts/AssetManager.sol:62]: don't understand, how user pay bridge fee(ETH) via off-chain when burn ?

**Comment by huwenchao** [comment]
merge to a separated branch first, check #263

**Pull request closed without merging**